### ISSUE TITLE
[Push Publishing] 'Title' field is updated with 'WidgetTitle' value once it's pushed.

### ISSUE THREAD

**Issue by joseorsini** [dotCMS/core#12618]
Push Publishing should create/update contents on the Target environment in the same way they were created/updated on the Source environment.

## Current Behavior
If we have a Widget-type content type with a 'Title' Field, and once content that has been created on the Source Environment is Pushed to another environments, the 'Title' field is populated with the value from Widget Title field IF the Title field is empty in the first place. 

## Known Workarounds
None.

## Steps to Reproduce (for bugs)

1. Set up a Push Publishing environment with two standalone servers. One as Source, one as Target.
2. On Source Environment, create a Test Widget content type as follows:

<img width="1411" alt="screen shot 2017-09-18 at 9 30 56 am" src="https://user-images.githubusercontent.com/1717110/30550642-3953779e-9c55-11e7-8318-3fdcf6e032e2.png">

Title field is not required and it can be empty once contents are created/updated.

3. Create a Test Content 001 on Source environment as follows: 

<img width="1416" alt="screen shot 2017-09-18 at 9 31 29 am" src="https://user-images.githubusercontent.com/1717110/30550792-c7f22fcc-9c55-11e7-89c1-cffe883b350e.png">

4. Push the content to Target environment.

<img width="1395" alt="screen shot 2017-09-18 at 9 31 47 am" src="https://user-images.githubusercontent.com/1717110/30550873-fcbefbc2-9c55-11e7-9a04-dede9a2509c7.png">

5. Check the content on Target environment. Title field has been populated with Widget Title field's values. 

<img width="1412" alt="screen shot 2017-09-18 at 9 32 10 am" src="https://user-images.githubusercontent.com/1717110/30550911-18fa3ee6-9c56-11e7-8fa9-fee61852f3bc.png">

## Context
Reproduced on latest version, as per customer's instructions https://my.dotcms.com/tickets/detail.dot?id=ffc912ec-fb6c-4234-a7d4-e586595849cd

## Your Environment

* dotCMS version used: 4.1.1.
* Browser Name and version: Any Browser.
* Operating System and version: Any OS.
* Application Server and version: Tomcat 8.0.18.
* Java Brand and version: Java 8u121.
* Database and version: H2, SQL Server.
* Application module (if apply): N/A

**Comment by joseorsini** [comment]
Update: 

After the content is pushed, the value on "Title" field remains empty on the receiver side, however:

- If we reindex all contents, "Title" field is going to be populated with the value of "widget Title" field.
- If we flush all caches, then the "Title" field is emptied out. 

This is how contents look in the DB in both sender and receiver environments: 

```
mysql> use dotcms411;
Reading table information for completion of table and column names
You can turn off this feature to get a quicker startup with -A

Database changed
mysql> select title, text1, text2 from contentlet where identifier = '513b31a1-caff-4993-a46f-5c84e5acd00b';
+------------------+------------------+-------+
| title            | text1            | text2 |
+------------------+------------------+-------+
| Test Content 001 | Test Content 001 | NULL  |
+------------------+------------------+-------+
1 row in set (0.00 sec)

mysql> use dotcms411receiver
Reading table information for completion of table and column names
You can turn off this feature to get a quicker startup with -A

Database changed
mysql> select title, text1, text2 from contentlet where identifier = '513b31a1-caff-4993-a46f-5c84e5acd00b';
+------------------+------------------+-------+
| title            | text1            | text2 |
+------------------+------------------+-------+
| Test Content 001 | Test Content 001 | NULL  |
+------------------+------------------+-------+
1 row in set (0.00 sec)

```

**Comment by nollymar** [comment]
PR Core: https://github.com/dotCMS/core/pull/12756

**Comment by dsilvam** [comment]
PR Unit test: https://github.com/dotCMS/core/pull/12778

**Comment by bryanboza** [comment]
Fixed, tested on the last release-4.2 // postgres // chrome. Now I am unable to reproduce this problem
